lib/pbench/agent/tool_data_sink.py

```diff
@@ -640,6 +642,12 @@ class BenchmarkRunDir:
     handle that seamlessly.
     """
 
+    class Exists(Exception):
+        pass
+
+    class Prefix(Exception):
+        pass
```

Comment:
It would be nice to have some clues as to the intended semantics of these exceptions.  As it stands, these are generic wrappers of a generic exception, without any message text or even code comments.

At a minimum, they should have explanatory code comments.  Beyond that, though, they could be derived from `NotADirectoryError` or `FileNotFoundError`.  Or, you could declare them to be instances rather than classes, and provide a message, like
```
Exists = ValueError("Benchmark run directory does not exist")
```

Reply:
Tracking in #2184.